brave/brave-core
# Brave rewards ui

Resolves https://github.com/brave/brave-browser/issues/6077

UI PR: https://github.com/brave/brave-ui/pull/540

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
Plan 1
- make sure that CI is green

Plan 2
- start browser
- enable rewards
- make sure that rewards page, panel and tip banner renders correctly

Plan 3
- make sure that `npm run storybook` works

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
